controllers/api/movie-routes.js

```diff
@@ -1,11 +1,15 @@
 const router = require('express').Router();
+const { Movie } = require('../../models');
 
 router.get("/movies", (req, res) => {
-    let results = books;
-    if (req.query) {
-      results = filterByQuery(req.query, results);
-    }
-    res.json(results);
+    Movie.findAll()
+    .then(movieData => {
```

Comment:
Is `movieData` supposed to be what the frontend will pass in to search off of? Like title/genre/etc.?

Reply:
So findAll is a promise based function, when you append that .then() it’s gonna receive an object returned by findAll as a parameter you can name that parameter anything in the callback function just as long as you maintain the same name